### PR TITLE
Made names on cards clickable issue#10

### PR DESCRIPTION
**Pull request by Jovani-JRH7**
### Description

Please include a summary of the changes and the related issue. 

Add issue number at the end
Issue: https://github.com/justsanchez/FESonline.github.io/issues/

### Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -514,8 +514,10 @@ <h2 class="titleModal">Yesmarie De La Flor</h2>
                 <img src="images/FJmorales.jpg" alt="Co President Javier Picture" loading="lazy">
             </a>
             <div class="card-info">
+                <a class="button open-modal" href="#javiPOPUP">
                 <h6>Javier Morales</h6>
                 <span>Co President</span>
+                </a>
```
Comment (justsanchez) [inline team.html:520]: The `<a>` tags should only surround the name not the role

File: team.html [hunk 2]
```diff
@@ -555,8 +559,10 @@ <h6>April Villeda</h6>
                 <img src="images/AJsanchez.jpg" alt="Vice President Alvaro Picture" loading="lazy">
             </a>
             <div class="card-info">
+                <a class="button open-modal" href="#alvaroPOPUP">
                 <h6>Alvaro Sanchez</h6>
                 <span>Vice President / Web Development Lead</span>
+                </a>
```
Comment (justsanchez) [inline team.html:565]: same here

File: team.html [hunk 3]
```diff
@@ -580,8 +586,10 @@ <h6>Alvaro Sanchez</h6>
                 <img src="images/Daniel(U).jpg" alt="Financial Advisor Daniel Picture" loading="lazy">
             </a>
             <div class="card-info">
+                <a class="button open-modal" href="#DaniMPOPUP">
                 <h6>Daniel Mendez-Perez</h6>
                 <span>Financial Advisor</span>
+                </a>
```
Comment (justsanchez) [inline team.html:592]: same here

File: team.html [hunk 4]
```diff
@@ -537,8 +539,10 @@ <h6>Javier Morales</h6>
                 <img src="images/Avilleda.jpg" alt="Co President Abril Picture" loading="lazy">
             </a>
             <div class="card-info">
+                <a class="button" href="#aprilPOPUP">
                 <h6>April Villeda</h6>
                 <span>Co President</span>
+                </a>
```
Comment (justsanchez) [inline team.html:545]: same here

File: team.html [hunk 5]
```diff
@@ -618,8 +626,10 @@ <h6>Daniel Mendez-Perez</h6>
                 <img src="images/newLesliePicture.jpg" alt="Social Media Coordinator Leslie Picture" loading="lazy">
             </a>
             <div class="card-info">
+                <a class="button open-modal" href="#LesliePOPUP">
                 <h6>Leslie Sanchez</h6>
                 <span>Social Media Coordinator</span>
+                </a>
```
Comment (justsanchez) [inline team.html:632]: same here

File: team.html [hunk 6]
```diff
@@ -637,8 +647,10 @@ <h6>Leslie Sanchez</h6>
                 <img src="images/yanni(U).jpg" alt="Secretary Ayanni Picture" loading="lazy">
             </a>
             <div class="card-info">
+                <a class="button open-modal" href="#AyanniPOPUP">
                 <h6>Ayanni Reese</h6>
                 <span>Secretary</span>
+                </a>
```
Comment (justsanchez) [inline team.html:653]: same here

**Review by justsanchez — CHANGES_REQUESTED** [review]

**Comment by justsanchez** [comment]
Also fill out the Pull Request info for every PR

**Comment by Jovani-JRH7** [comment]
Sorry about that, I'm used to pressing CTRL + Enter to jump to the next line. When I tried doing that when creating the pull request it would just submit it instead of jumping to the next line.
Won't happen again, I'll fill out the pull request info next time.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Jovani-JRH7** [comment]
Have fixed errors.

**Review by justsanchez — APPROVED** [review]

**Pull request merged**